components/Trial/Form.js

```diff
@@ -33,6 +39,23 @@ const styles = {
     marginTop: 40,
     width: 170,
     textAlign: 'center'
+  }),
+  successActions: css({
+    display: 'flex',
+    flexDirection: 'column',
+    [mediaQueries.mUp]: {
+      display: 'inherit'
+    }
```

Comment:
Alternative solution:
```suggestion
    },
    '& :first-child': {
      margin: '0 0 10px'
    },
    [mediaQueries.mUp]: {
      '& :first-child': {
        margin: '0 10px 0 0'
      }
    }
```
Then we also don't need prop forwarding on button.

Reply:
Duly noted!